platform/linux-generic/odp_packet_io.c

```diff
@@ -150,6 +150,8 @@ static void init_pktio_entry(pktio_entry_t *entry)
 	init_out_queues(entry);
 
 	pktio_classifier_init(entry);
+
+	odp_pktio_config_init(&entry->s.config);
```

Comment:
It seem odp_pktio_config_init() is now called twice for the same entry as it's already called in setup_pktio_entry().